src/components/ChatLog.jsx

```diff
@@ -0,0 +1,40 @@
+import ChatEntry from './ChatEntry';
+import './ChatLog.css';
+import PropTypes from 'prop-types';
+
+
+const ChatLog = ({entries,onItemLikedToggle}) => {
```

Comment:
Nitpick: fix up whitespacing

In industry, when you're creating a pull request for review, be sure that your code is neat, the style conforms to the style guide, stray comments are removed. Typically, a teammate will not review a PR until the code styling issues are fixed up. 

Code with styling issues are not allowed to be merged into main because it would introduce inconsistencies in a codebase that compromises readability and the maintenance of a project.

```suggestion
const ChatLog = ({entries, onItemLikedToggle}) => {
```